### ISSUE TITLE
Angular 10 & 11 component not working when using routing

### ISSUE THREAD

**Issue by hugochouinard** [ng-bootstrap/ng-bootstrap#3924]
### Bug description:

I am only able to make components (like Tooltips and Nav) work in the main component of an application.  I had no success when trying to use the components in any child component or lazy loaded modules.  I thought it did not work with Angular 11 yet but I did the StackBlitz with Angular 10 and I get the same result.

### Link to minimally-working StackBlitz that reproduces the issue:

[StackBlitz](https://stackblitz.com/edit/angular-ivy-mfnuba?file=src/app/app.component.html) The tooltip when hovering the main button is working but not with the button in the customers component.

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 10 & 11

ng-bootstrap: 8.0.0

Bootstrap: 4.5.0

**Comment by jnizet** [comment]
This is not an issue with ng-bootstrap. You should ask such questionson stackoverflow.

Your stackblitz doesn't use your customers module anywhere, doesn't use lazy loading at all, and forgets to declare the customers component, which is used in the root module, in this root module.

Here's a stackblitz where the customers component is properly declared in the root module. It works fine: https://stackblitz.com/edit/angular-ivy-9atruh?file=src%2Fapp%2Fapp.module.ts.

Here's a stackblitz where the root module imports the customers module, and where the customers module properly imports the NgbModule (since its customers component uses ng-bootstrap): https://stackblitz.com/edit/angular-ivy-emcsx4?file=src%2Fapp%2Fcustomers%2Fcustomers.module.ts. It works fine.

Here's a stackblitz where the customers module is loaded lazily, and where it properly imports NgbModule (since its customers component uses ng-bootstrap): https://stackblitz.com/edit/angular-ivy-5qpxf9?file=src%2Fapp%2Fapp-routing.module.ts. It works fine, too. 

So, to recap: if a component is part of a module, it must be declared in that module. If a component of a module uses ng-bootstrap, then that module must import NgbModule.

**Comment by hugochouinard** [comment]
Ok my bad!  I thought I could load the NgbModule in app.module.ts even with lazy loading but you're right!  Now I tested with Angular 11 and it is working fine! Thanks a lot @jnizet
